data/net/minecraft/core/Vec3i.mapping

```diff
@@ -10,14 +11,14 @@ CLASS net/minecraft/core/Vec3i
 		ARG 2 y
 		ARG 3 z
 	METHOD above ()Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector 1 block up
+		COMMENT Creates a new Vec3i one unit up.
 	METHOD above (I)Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector upwards by the given distance.
+		COMMENT Creates a new Vec3i up by the distance.
```

Comment:
This is technically incorrect, since if the distance is `0`, then it returns the same vector object.

Reply:
Any idea on how to phrase it, so it's clear it creates a new Vec3i if the distance is not 0, without being too long?

Reply:
```
If distance is not 0, will create a new Vec3i offset by the distance upwards, otherwise will return this vector.
```
something like that maybe?